src/open_dread_rando/door_locks/custom_door_types.py

```diff
@@ -19,30 +19,34 @@
 ENERGY_MP_MAT = "actors/props/door_shield_plasma/models/imats/doorshieldplasma_mp_opaque_01.bsmat"
 ENERGY_FX_MAT = "actors/props/door_shield_plasma/models/imats/doorshieldplasma_matfx.bsmat"
 SMOOTH_NORMALS = "actors/props/doorshieldmissile/models/textures/shield_no_normals.bctex"
```

Comment:
I'd rather update this than change the file name. or change the shield_no_attribs_at.bctex in the same way.